src/lang/de.php

```diff
@@ -125,3 +125,7 @@
 $lang['logsSearch'] = 'Suche';
 $lang['logsPrivateChannels'] = 'Private Räume';
 $lang['logsPrivateMessages'] = 'Private Nachrichten';
+$lang['helpItemDescClear'] = 'Alle Nachrichten lÃ¶schen (hilft auch wenns mal haken sollte)';
```

Comment:
Please pay attention to the settings of the character set UTF-8.

Reply:
Info:

/clear
https://github.com/Frug/AJAX-Chat/issues/63
https://github.com/Frug/AJAX-Chat/pull/64

/me
https://github.com/Frug/AJAX-Chat/blob/d2fcf98e1a1147b69181f10bdb8c4ce0c7590588/chat/changelog.txt#L257


https://github.com/Frug/AJAX-Chat/blob/d2fcf98e1a1147b69181f10bdb8c4ce0c7590588/chat/changelog.txt#L572

> - Removed the /me command as it had no real use.

In version 0.7 von 2007 /me should be removed for good. It was already seen as unnecessary at the time. Apparently it was accidentally reinserted by a code change

Reply:
To me, /me is not unnecessary, I will leave it. Sorry for the wrong encoding, for some reason my editor was set to convert automatically.
I overlooked that ;)

Reply:
/me and /action is the is the same. 2 identical commands are not required. A css value is already available with action. This command can be designed with css:

https://github.com/wintstar/AJAX-Chat/blob/main/public/css/fonts.css#L74

A message text must also be entered with the command. Otherwise an error message will appear. This is already available with the language strings for /action:

> Describe action: /action Text 

/me is removed.